<!-- -*- mode: XML -*- -->
<!-- run velodyne_pointcloud/CloudNodelet in a nodelet manager -->

<launch>
  <arg name="calibration" default="" />
  <arg name="device_model" default="" />
  <arg name="manager" default="velodyne_nodelet_manager" />
  <arg name="max_range" default="200.0" />
  <arg name="min_range" default="0.9" />

  <node pkg="nodelet" type="nodelet" name="$(arg manager)_cloud"
        args="load velodyne_pointcloud/CloudNodelet $(arg manager)">
    <param name="calibration" value="$(arg calibration)"/>
    <param name="device_model" value="$(arg device_model)" />
    <param name="max_range" value="$(arg max_range)"/>
    <param name="min_range" value="$(arg min_range)"/>
  </node>
</launch>
